src/node.ts

```diff
@@ -85,7 +85,12 @@ export class EthereumNode extends constructs.Construct {
 
     super(scope, id);
 
+
     const region = cdk.Stack.of(this).region;
+    const availabilityZone = cdk.Stack.of(this).availabilityZones;
+    // In order to support Environment Agnostic Stacks (https://docs.aws.amazon.com/cdk/v2/guide/environments.html),
```

Comment:
I don't think the explanation here is needed when the variable is more clearly named.

Reply:
Removed